src/netex_idf/lines.rs

```diff
@@ -63,12 +68,43 @@ fn line_color(line: &Element, child_name: &str) -> Option<Rgb> {
         .and_then(|p| p.only_child(child_name)?.text().parse().ok())
 }
 
+pub fn get_or_create_trip_property<'a>(
+    line: &Element,
+    trip_propertiess: &'a mut HashMap<Accessibility, TripProperty>,
+) -> Option<&'a mut TripProperty> {
+    let accessibility_node = line.only_child("AccessibilityAssessment")?;
+    let id: String = accessibility_node.attribute("id")?;
+    let accessibility = accessibility(accessibility_node)?;
+
+    let trip_property = trip_propertiess
+        .entry(accessibility.clone())
+        .or_insert_with(|| {
+            let Accessibility {
+                wheelchair,
```

Comment:
You can destructure and rename the variable.
```suggestion
                wheelchair: wheelchair_accessible,
```